### ISSUE TITLE
pip install when building a docker image takes too long

### ISSUE THREAD

**Issue by id5h** [dgoeries/lttbc#24]
Hi, I need to install lttbc when building a python docker image, based on python:3.11.4-slim-bookworm.

The command in the Dockerfile is:
`RUN pip3 install --no-cache-dir lttbc==0.2.4`

However, it takes more than 10 minutes to finish installing the build dependencies 
(I get the message `Installing build dependencies: still running...`).
I do not see any dependencies for lttbc in the Readme.

For completeness, I also `RUN apt-get install -y build-essential libblas-dev gcc` before running the pip install stage.
I am on an M1-pro and building the image with the flag `--platform linux/amd64`.

Any ideas what might cause this problem?

**Comment by dgoeries** [comment]
I believe it will try to install numpy version 1.22 for the build process. This problem occured when switching to the toml structure. Putting a build dependency specification with numpy>=1.22 was always taking the highest available, e.g. 1.24. This did result in compatibility problems.

I could provide another lttbc with a different toml to circumvent this

**Comment by dgoeries** [comment]
E.g. use 1.24 for the build process

**Comment by id5h** [comment]
Thanks for the prompt reply @dgoeries !

I am running now with `RUN pip3 install --no-cache-dir numpy==1.22 lttbc==0.2.4`, but see no difference

**Comment by id5h** [comment]
Same with `RUN pip3 install --no-cache-dir numpy==1.24 lttbc==0.2.4`.

Did I misunderstand your suggestion?

**Comment by dgoeries** [comment]
Yes, it will still try to install numpy=1.22 for the build process :(. I need to prepare a new version and document
